battleship-server/src/main/java/com/academy/solid/nie/server/Main.java

```diff
@@ -1,34 +1,22 @@
 package com.academy.solid.nie.server;
 
-import java.io.IOException;
-import java.util.logging.Logger;
+import java.util.stream.IntStream;
 
 /**
  * Main class for the game.
  */
 public final class Main {
-    private static final Logger LOGGER = Logger.getLogger(Main.class.getName());
-
     private Main() {
     }
 
     /**
      * Creates server here.
      *
      * @param args represents input
-     * @throws IOException when something happens with the Server.
      */
-    public static void main(final String[] args) throws IOException {
-        GameInitializer gameInitializer;
-        Player first = new NetPlayer();
-        Player second = new NetPlayer();
-        int port = Integer.parseInt(args[0]);
-        gameInitializer = new ServerGameInitializer(first, second, port);
-        gameInitializer.initializeGame();
-        Game game = new Game(first, second);
-        while (!game.isGameOver()) {
-            game.play();
-        }
-        LOGGER.info("Game over");
+    public static void main(final String[] args) {
+        int firstPort = 8080;
```

Comment:
we need way to configure those like number of games and based on that we need to count ports. But I wonder Are you sure that we need many ports?